src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double firstLeg, double secondLeg, double height) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstLeg + secondLeg) * height) / 2;
```

Comment:
The formula for the area of an isosceles trapezoid is incorrect. The correct formula is the average of the lengths of the two parallel sides (the bases) multiplied by the height, which is `((firstLeg + secondLeg) / 2) * height`. Please adjust the formula accordingly.